src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -114,6 +114,11 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// </summary>
         internal FeatureFilterTracing FeatureFilterTracing { get; set; } = new FeatureFilterTracing();
 
+        /// <summary>
+        /// Options used to configure provider startup.
+        /// </summary>
+        public StartupOptions Startup { get; set; } = new StartupOptions();
```

Comment:
I notice that ClientOptions is internal and we have `ConfigureClientOptions`. Seems we should stick with that pattern given we have established it?

Reply:
`ConfigureStartupOptions` sounds like a good idea